src/endpoints/handlers.hpp

```diff
@@ -3,31 +3,38 @@
 
 #include "common.hpp"
 
+#ifndef IRODS_HTTP_API_ENDPOINT_ENTRY_FUNCTION_SIGNATURE
```

Comment:
why is this in an ifndef ?    is this something we're ever going to be interested in *not* having/using?

seems helpful/good.

Reply:
got it.